### PR TITLE
Fix for Issue 245 and refactored code in usecase

### LINKED ISSUE #245: LIG is not getting created with uplinkset of networkType FCNetwork
### Scenario/Intent
LIG is not getting created with uplinkset of networkType FCNetwork

### Environment Details
 - **Puppet module for HPE OneView Version:** [Version of this module for which you are encountering the issue]
 - **OneView SDK Version:** 5.0
 - **OneView Client API Version:** 1200
 - **Puppet Version:** 5.5.0
 - **Ruby Version:** 2.4.x
 - **Platform:** Ubuntu

### Steps to Reproduce
Create a Logical Interconnect Group resource with uplinkset of networkType FCNetwork.

### Expected Result
It should successfully create Logical Interconnect Group with all the conditions given while creating resource.

### Actual Result
Error is - FCNetwork: uninitialized constant

### PR DESCRIPTION
**Pull request by chebroluharika**
### Description
Fix for Issue 245 and refactored code in usecase

### Issues Resolved
#245

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Review by AnushaY1916 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# v2.6.0
```
Comment (soodpr) [inline CHANGELOG.md]: the version should be 2.5.1 since its just a bug fix.

Comment (chebroluharika) [reply CHANGELOG.md]: later changes will be there to support API1600. Even then do I need to change version?

Comment (soodpr) [reply CHANGELOG.md]: i would say when that change comes, then change the version. Lets follow the semantic versioning rules for now.

Comment (chebroluharika) [reply CHANGELOG.md]: done

**Review by soodpr — APPROVED** [review]
LGTM 👍

**Pull request merged**